frontend/src/App.css

```
@@ -37,6 +57,178 @@ nav a:hover {
+  background: linear-gradient(135deg, #667eea 0%, #764ba2 100%);
+  color: white;
+  padding: 16px 32px;
+  border-radius: 30px;
+  font-size: 18px;
+  font-weight: 600;
+  box-shadow: 0 4px 20px rgba(102, 126, 234, 0.4);
+  z-index: 1000;
+  animation: fadeInOut 4s ease forwards;
+}
+
+@keyframes fadeInOut {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeInOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)